cfme/tests/containers/test_containers_projects_summary.py

```diff
@@ -14,11 +14,18 @@
 pytest_generate_tests = testgen.generate(
     testgen.container_providers, scope='function')
 
-
 projects_properties_fields = ['Name', 'Creation timestamp', 'Resource version']
-projects_relationships_fields = ['Containers Provider', 'Routes', 'Services', 'Replicators',
-                                 'Pods', 'Nodes']
+projects_relationships_fields_lowest = ['Containers Provider', 'Routes', 'Services', 'Replicators',
+                                        'Pods', 'Nodes']
+projects_relationships_fields_57 = ['Containers Provider', 'Routes', 'Container Services',
+                                    'Replicators', 'Pods', 'Nodes']
+
+projects_relationships_fields_key = ({
+    version.LOWEST: projects_relationships_fields_lowest,
+    '5.7': projects_relationships_fields_57
+})
 
+projects_relationships_fields = version.pick(projects_relationships_fields_key)
```

Comment:
`version.pick` would be ran on module import - this is undesirable. Either do the version picking inside the test where you need it or if it's needed in multiple tests, define it as a function; example here: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/infrastructure/test_host.py#L22